jujugui/static/gui/src/app/components/profile/header/header.js

```diff
@@ -0,0 +1,22 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+class ProfileHeader extends React.Component {
```

Comment:
Docstring for the component, how is it used, for what purpose, what does it provide?